apps/cow-tools/src/app/milkman/utils/type.ts

```diff
@@ -1,4 +1,7 @@
 export enum TransactionStatus {
+  DRAFT_SELECT_TOKENS = "Draft: Select tokens",
+  DRAFT_SELECT_PRICE_CHECKER = "Draft: Select price checker",
+  DRAFT_RESUME = "Draft: Resume",
```

Comment:
I would separate this into a different enum object

Reply:
didn't understand